src/mpclookup/handlers/external.py

```diff
@@ -60,22 +61,48 @@ async def get_index(
 async def search(
     designation: Annotated[str, Query()],
     logger: Annotated[BoundLogger, Depends(logger_dependency)],
+    request: Request,
 ) -> str:
     """
-    Request a redirect to the MPCORB record for a given designation.
+    Request a redirect to the MPCORB database record for a given designation.
 
     Notes
     -----
-    Example request:
+    An example request could be:
 
-    /mpc-lookup/search?designation=2011+1001+T-2
+    `/mpc-lookup/search?designation=2011+1001+T-2`
 
+    The "2011 " prefix, which is present in the DP0.3 data, will be
+    automatically stripped out so that the designation has a valid format.
+
+    A designation without any spaces will cause a redirect to an endpoint
+    which returns a message indicating that the object appears to be synthetic.
     """
     logger.info("Request for designation URL", designation=designation)
     fd = designation.replace(_DESIGNATION_PREPEND, "")
-    redirect_url = (
-        "https://www.minorplanetcenter.net/db_search/"
-        f"show_object?object_id={fd}"
-    )
-    logger.info("Redirecting to designation URL", redirect_url=redirect_url)
+    if " " in fd:
+        redirect_url = urlparse(
+            "https://www.minorplanetcenter.net/db_search/"
+            f"show_object?object_id={fd}"
+        ).geturl()
+        logger.info("Redirecting to MPC URL", redirect_url=redirect_url)
+    else:
+        redirect_url = urlparse(
+            str(request.url_for("get_synthetic_object"))
+            + "?"
+            + urlencode({"designation": designation})
+        ).geturl()
+        logger.info(
+            "Redirecting synthetic object to", redirect_url=redirect_url
+        )
     return redirect_url
+
+
+@external_router.get("/synthetic_object", response_class=PlainTextResponse)
```

Comment:
This needs to be an `HTMLResponse` in order to match the "text/html" MIME type assigned to the service descriptor.

Reply:
This should be fixed now.